common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -38,13 +40,22 @@ class SequenceFields(object):
     # NOTE: Position is 1-indexed.  This is silly, but there are now student
     # positions saved on prod, so it's not easy to fix.
     position = Integer(help="Last tab viewed in this sequence", scope=Scope.user_state)
+
     due = Date(
         display_name=_("Due Date"),
         help=_("Enter the date by which problems are due."),
         scope=Scope.settings,
     )
 
-    # Entrance Exam flag -- see cms/contentstore/views/entrance_exam.py for usage
```

Comment:
Should this docstring stay with `is_entrance_exam` below, or was it intentionally removed?